src/pages/new/volunteer/index.js

```diff
@@ -1,7 +1,31 @@
 import React from "react";
 
+import ImageHeader from "../../../components/grid-aware/ImageHeader";
+import classroom from "../../../components/grid-aware/ImageHeader/stories/classroom.png";
+import whiteboard from "../../../components/grid-aware/ImageHeader/stories/whiteboard-sticky-notes.png";
 import Layout from "../../../components/layout";
 
 export default () => (
-  <Layout>This is a placeholder for the volunteer page.</Layout>
+  <Layout>
+    <ImageHeader
+      title="Volunteering"
+      subtitle="We rely on hundreds of volunteers to bridge the digital divide that our most vulnerable neighbors face."
+      description="If you share our belief that digital equity is a human right, we encourage you to apply and work with us."
+      image1={{
+        url: classroom,
+        alt: "Classroom of volunteer members collaborating with one another.",
+      }}
+      image2={{
+        url: whiteboard,
+        alt:
+          "Team members collaborating together by looking and pointing at whiteboard covered with sticky notes full of ideas.",
+      }}
+      ctaButtons={[
+        {
+          text: "APPLY",
```

Comment:
Text transform to uppercase is inherent when passing the text prop to a Button component.